<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditCtUserUi</class>
 <widget class="QWidget" name="EditCtUserUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Crypt Token User</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="TextLabel1_5_2_3">
     <property name="text">
      <string>&lt;qt&gt;
&lt;p&gt;Please enter the bank and user data.&lt;/p&gt;
&lt;/qt&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>User number on medium</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="userCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>The index number of the current user on the chosen medium, if the medium can contain more than one user.</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer31">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>231</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Bank</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="TextLabel1_2_2_2_2_2_2_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Bank Code</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLineEdit" name="bankCodeEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Enter the bank code (BLZ) of your bank</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bankCodeButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="TextLabel1_4_2_3_2_2_2_2_2_2">
        <property name="text">
         <string>Bank Name</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="bankNameLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="TextLabel1_4_2_3_2_2_2_2_2">
        <property name="text">
         <string>Server</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="serverEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Enter the address of the bank server. For PIN/TAN, this is a full URL with server address and a trailing path.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>User</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="TextLabel1_6_2_2_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nameEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Enter the name of the user. This is needed as originator name when an online transfer job is executed.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="TextLabel1_3_2_2_2_3">
        <property name="text">
         <string>User Id</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="userIdEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Enter the user id that the bank has given to you.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="TextLabel1_3_2_2_2_2_2">
        <property name="text">
         <string>Customer Id</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="customerIdEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Enter the customer id, if the bank has given one to you. If the bank has not given one to you, then leave this field blank.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showSpecialCheck">
     <property name="text">
      <string>Show Special Settings</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="specialGroup">
     <property name="title">
      <string>Special Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="textLabel1_2_2">
          <property name="text">
           <string>HTTP Version</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="httpVersionCombo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Select the HTTP version to be used in PIN/TAN mode</string>
          </property>
          <item>
           <property name="text">
            <string>1.0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1.1</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="2" colspan="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>28</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="textLabel1_2">
          <property name="text">
           <string>HBCI Version</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QComboBox" name="hbciVersionCombo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>HBCI 2.01</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>HBCI 2.10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>HBCI 2.20</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>HBCI 3.00</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="textLabel2">
          <property name="text">
           <string>RDH Version</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="rdhVersionCombo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Select the HTTP version to be used in PIN/TAN mode</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="TextLabelDescription">
          <property name="text">
           <string>Bank Key Id</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="5">
         <widget class="QLineEdit" name="peerIdEdit">
          <property name="toolTip">
           <string>Key id used by the bank. If left blank then the user id is used</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="6">
         <widget class="Line" name="line1">
          <property name="frameShape">
           <enum>QFrame::HLine</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QCheckBox" name="bankSignCheck">
          <property name="text">
           <string>Bank signs its messages</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3" colspan="2">
         <widget class="QCheckBox" name="forceSsl3Check">
          <property name="toolTip">
           <string>Force the use of SSLv3</string>
          </property>
          <property name="whatsThis">
           <string>&lt;qt&gt;Force the use of &lt;b&gt;SSLv3&lt;/b&gt; in &lt;b&gt;Pin/Tan&lt;/b&gt; mode&lt;/qt&gt;</string>
          </property>
          <property name="text">
           <string>Force SSLv3</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="3">
         <widget class="QCheckBox" name="bankCounterCheck">
          <property name="text">
           <string>Bank uses signature counter</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3" colspan="3">
         <widget class="QCheckBox" name="noBase64Check">
          <property name="toolTip">
           <string>Force the use of SSLv3</string>
          </property>
          <property name="whatsThis">
           <string>&lt;qt&gt;Force the use of &lt;b&gt;SSLv3&lt;/b&gt; in &lt;b&gt;Pin/Tan&lt;/b&gt; mode&lt;/qt&gt;</string>
          </property>
          <property name="text">
           <string>No BASE64 Encoding</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>userCombo</tabstop>
  <tabstop>bankCodeEdit</tabstop>
  <tabstop>bankCodeButton</tabstop>
  <tabstop>showSpecialCheck</tabstop>
  <tabstop>httpVersionCombo</tabstop>
  <tabstop>rdhVersionCombo</tabstop>
  <tabstop>hbciVersionCombo</tabstop>
  <tabstop>peerIdEdit</tabstop>
  <tabstop>bankSignCheck</tabstop>
  <tabstop>bankCounterCheck</tabstop>
  <tabstop>forceSsl3Check</tabstop>
  <tabstop>noBase64Check</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
